### ISSUE TITLE
PHP version doesn't work, or am I missing anything?

### ISSUE THREAD

**Issue by bago** [usrflo/registered-domain-libs#8]
require_once("registered-domain-libs/effectiveTLDs.inc.php");
require_once("registered-domain-libs/regDomain.inc.php");

getRegisteredDomain('subdomain.domain.co.uk'); 
// this print "co.uk"
getRegisteredDomain('subdomain.domain.co.uk', $tldTree); 
// this print "co.uk"

**Comment by bago** [comment]
I fixed this by adding this line:

```php
        /* load tld tree into object */
        function __construct() {
                /* include tld tree data */
                include(dirname(__FILE__) . '/effectiveTLDs.inc.php');
+                $this->tldTree = $tldTree;
        }
```

**Comment by jakubsuchy** [comment]
Works for me too

**Comment by giosh94mhz** [comment]
Should be closed as it already fixed by commit bd9b7dee7d7f4e61bdaac1980fc9ad17f7478155
